.changeset/cold-humans-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-icons-webc": minor
+---
+
+[Removed] - exports section in package.json to make consumer experience more consistent
```

Comment:
I definitely understand wanting to make the experience more consistent – I just wonder if there's a way to cater for both (while providing this info in the docs depending on if people are using Webpack 4 or 5).

Mainly as it is so much nicer to be able to drop file extensions when importing (and that's a fairly common expectation for module imports).

Reply:
Part of this is actually _forcing_ people to use file extensions!

I thought I wrote quite a long description for this PR explaining it but it must have gotten lost... 😢

Reply:
For further reading, see Lit's recommendation here: https://lit.dev/docs/tools/publishing/#include-file-extensions-in-import-specifiers

And also you can read the description and AC for DSW-1777, which addresses this directly

Reply:
That makes sense - completely ignore me then 😀 if the best practices say that then let's do it for sure.

Are we planning on documenting our import recommendation somewhere in our docs/wiki (mainly to point to this Lit best practice)?

Reply:
Yes, I'll be updating the docs as part of this ticket! (Might end up being a separate PR though)